frontend/src/features/properties/filter/PropertyFilter.test.tsx

```diff
@@ -25,7 +25,7 @@ jest.mock('actionCreators/propertyActionCreator');
 
 const mockedAxios = axios as jest.Mocked<typeof axios>;
 const mockStore = configureMockStore([thunk]);
-const history = createMemoryHistory();
+let history = createMemoryHistory();
```

Comment:
history needs to be reset at the start of every test or the filter bar state will persist.